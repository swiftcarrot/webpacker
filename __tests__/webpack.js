const uglify = require('../src/webpack/uglify');

test('uglify', () => {
  expect(uglify).toMatchSnapshot();
});
