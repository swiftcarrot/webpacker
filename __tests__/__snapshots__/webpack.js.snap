// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uglify 1`] = `
Object {
  "compress": Object {
    "dead_code": true,
    "unused": true,
    "warnings": false,
  },
  "minimize": true,
  "output": Object {
    "comments": false,
  },
  "sourceMap": true,
}
`;
